modules/thermal_hydraulics/test/tests/jacobians/materials/ad_solid_material.i

```diff
@@ -0,0 +1,88 @@
+[Mesh]
+  type = GeneratedMesh
+  dim = 1
+  nx = 3
+  allow_renumbering = false
+[]
+
+[Variables]
+  [T]
+  []
+[]
+
+[Functions]
+  [k_fn]
+    type = ParsedFunction
+    value = 't*t + 2*t'
+  []
+
+  [cp_fn]
+    type = ParsedFunction
+    value = 't*t*t + 3*t'
+  []
+
+  [rho_fn]
+    type = ParsedFunction
+    value = 't*t*t*t + 4*t'
+  []
+[]
+
+[HeatStructureMaterials]
+  [prop_uo]
+    type = SolidMaterialProperties
+    k = k_fn
+    cp = cp_fn
+    rho = rho_fn
+  []
+[]
+
+[Components]
```

Comment:
Is that empty block needed?

Reply:
Yes, that's what triggers the instantiation of `THMProblem` and allows us to use THM syntax like `HeatStructureMaterials`.